h8mir-BJS2-88187

Deathroels opened the pull request.

Deathroels committed:
BJS2-88187:
-completed 1&2 task items

Deathroels committed:
BJS2-88187:
-complete tasks items
-add List validation method;

MarselYsup commented on src/main/java/school/faang/bjs2_88187/ListOperations.java:
```diff
@@ -0,0 +1,77 @@
+
+    public static long countStringsStartingWith(List<String> strings, char simbol) {
+        return strings.stream()
+
```
лишняя пустая строка

MarselYsup commented on src/main/java/school/faang/bjs2_88187/ListOperations.java:
```diff
@@ -0,0 +1,77 @@
+                .allMatch(condition);
+    }
+
+    public static int findMinGreaterThan(List<Integer> numbers, int number) {
```
number - слишком общее название для аргумента, являющийся пороговым значением - тут можно переимновать на threshold

MarselYsup commented on src/main/java/school/faang/bjs2_88187/ListOperations.java:
```diff
@@ -0,0 +1,77 @@
+                .orElseThrow(() -> new NoSuchElementException("Пусто"));
+    }
+
+    public static long countStringsStartingWith(List<String> strings, char simbol) {
```
symbol*

Deathroels committed:
BJS2-88187: refactor
-work with review comments
-delete other comments
-delete validate code duplicate
